.gitignore

```diff
@@ -5,11 +5,11 @@
 .idea/
 .*.swp
 .*.un~
+*.iml
 
 # NPM artifacts
 node_modules/
 npm-debug.log
```

Comment:
Should ignore npm-*; there's a cache npm drops in the cwd (I think depending on your deps).